.github/workflows/snapshot-release.yaml

```diff
@@ -0,0 +1,88 @@
+          yq -ei '.metadata.version=env(SHORT_SHA), (.packages[]|select(has("ref"))|select(.name=="core-slim-dev")).ref=env(SHORT_SHA)' bundles/k3d-slim-dev/uds-bundle.yaml
+          yq -ei '.metadata.version=env(SHORT_SHA)' packages/slim-dev/zarf.yaml
+          yq -ei '.metadata.version=env(SHORT_SHA)' packages/standard/zarf.yaml
+
+      - name: Create Packages and Bundles
+        run: |
+          ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml standard-package --no-progress --set FLAVOR=${{ matrix.flavor }}
+          ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml k3d-standard-bundle --no-progress
+          ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml slim-dev-package --no-progress --set FLAVOR=${{ matrix.flavor }}
+          ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml k3d-slim-dev-bundle --no-progress
+
+          if [ "${{ matrix.flavor }}" != "registry1" ]; then
+            ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml standard-package --no-progress --set FLAVOR=${{ matrix.flavor }}
+            ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml k3d-standard-bundle --no-progress
+            ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml slim-dev-package --no-progress --set FLAVOR=${{ matrix.flavor }}
+            ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml k3d-slim-dev-bundle --no-progress
+          fi
+
+      # Standard Package by default tests what's in the Istio Package
+      #- name: Test amd64 Bundle
+      #  run: |
+      #    uds run -f tasks/deploy.yaml k3d-standard-bundle --no-progress
+      #    uds run -f tasks/test.yaml validate-packages --no-progress
+
+      - name: Publish Standard Package
+        run: uds run -f tasks/publish.yaml standard-package --set FLAVOR=${{ matrix.flavor }} --set TARGET_REPO="ghcr.io/defenseunicorns/packages/uds/snapshots" --set VERSION="${SHORT_SHA}"
+
+      - name: Publish Upstream Flavored Bundles
+        if: ${{ matrix.flavor != 'registry1'}}
+        run: uds run -f tasks/publish.yaml bundles --set TARGET_REPO="ghcr.io/defenseunicorns/packages/uds/snapshots" --set VERSION="${SHORT_SHA}"
+
+      - name: Save logs
+        if: always()
+        uses: ./.github/actions/save-logs
+        with:
+          suffix: -${{ matrix.flavor }}
+
+
+  update-snapshot-tag:
+    needs: publish-snapshot-release
+    env:
+      GH_TOKEN: ${{ github.token }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Update snapshot-latest tag
+        run: |
+          if gh release view snapshot-latest; then
+            gh release delete snapshot-latest --cleanup-tag -y
+          fi
+          gh release create snapshot-latest -t "snapshot-latest" -n "" --prerelease
```

Comment:
Not sure I like having the release/git-tag named `snapshot-latest` but the published OCI artifact using the SHA. I'm not opposed to either one, but they aren't aligned then its not obvious to an end user what OCI artifact to consume to get the latest snapshot (unless I'm missing some way these are linked). _Yes_ the sha does show on the release page but if we don't make it very easy to just grab the snapshot I fear that few people would? This would also result in a lot of "orphaned" OCI artifacts that aren't referenced from a release.

Couple options:
- go all in on `snapshot-latest`, tags+OCI: downside here is that the snapshot gets overwritten nightly/whenever, but not sure we care about that?
- go all in on sha, tags+OCI: downside of lots of releases, harder to clean these up + lots of release artifacts (OCI)
- hybrid...
  - oci tags: both sha and a `latest` equivalent
  - git tags/releases: only latest
  - provides history of all snapshot without cluttering releases, but does anyone care to pull an old snapshot if they don't know about it?

I don't generally mind using the sha in our naming but I think we need to be careful about cluttering our releases and OCI artifacts by doing these snapshots and not cleaning them up in a responsible way.

Reply:
Discussed briefly synchronously - path forward is along these lines:
- OCI publishing:
  - publish to different snapshot path
  - publish every snapshot (nightly/adhoc) without overwriting old ones
  - maybe some cleanup job to wipe out anything older than a week / the latest minor release (pending discussion on value/cost here)
  - tags for these would be sha+some date/timestamp
  - also tag the latest one with snapshot-latest
- Releases:
  - no releases per sha
  - single snapshot-latest release that is updated everytime we cut a new snapshot
  - basic release notes diffing what is included (diff between the snapshot and most recent real release?) - not 100% required, but nice to have

Reply:
will plan to open a follow up issue (if this change is merged) for cleaning up snapshots.  otherwise, all these items should be addressed now in the latest changes.